### ISSUE TITLE
fixed-width tabs

### ISSUE THREAD

**Issue by luin** [connors/photon#25]
Thank you for the photon! I've spent several hours to implement my own tab styles until I find this awesome library.

Would you like to add a fixed-width `tab-item` which can be found in many native apps (e.g. Terminal & Xcode):

![image](https://cloud.githubusercontent.com/assets/635902/10508152/cd4df560-7357-11e5-89bf-1b9620785856.png)
![image](https://cloud.githubusercontent.com/assets/635902/10508273/aa87d57c-7358-11e5-95ce-f2d43e0846bb.png)

I can submit a pr for this.

**Comment by developit** [comment]
Something like this?
http://jsfiddle.net/developit/tkx4r52s/

**Comment by luin** [comment]
Exactly. Just wonder whether it should be a built-in style though it's trivial to implement it.
